app/forms/ctc/account_number_form.rb

```diff
@@ -3,7 +3,7 @@ class AccountNumberForm < QuestionsForm
     set_attributes_for :bank_account, :account_number
     set_attributes_for :confirmation, :account_number_confirmation
 
-    validates :account_number, confirmation: true, presence: true
+    validates :account_number, presence: true, confirmation: true, length: { maximum: 17 }, account_number: true
```

Comment:
i dont... im sure there's some reasonable minimum, but i couldnt find anything definitive online about it. i could probably add something like 5, as it would be absurd to have a bank account less than 5 digits.

Reply:
I'm fine with no minimum in this case!